tests/test_parse.c

```diff
@@ -149,10 +149,10 @@ static void test_utf8_parse()
 	// json_tokener_parse doesn't support checking for byte order marks.
 	// It's the responsibility of the caller to detect and skip a BOM.
 	// Both of these checks return null.
-	char utf8_bom[] = {0xEF, 0xBB, 0xBF, 0x00};
-	char utf8_bom_and_chars[] = {0xEF, 0xBB, 0xBF, '{', '}', 0x00};
-	single_basic_parse(utf8_bom, 0);
-	single_basic_parse(utf8_bom_and_chars, 0);
+	unsigned char utf8_bom[] = {0xEF, 0xBB, 0xBF, 0x00};
```

Comment:
Instead of changing the type and casting here, would it work to use e.g. "\xef\xbb\xbg" instead?

Reply:
I'll give it a go.

Reply:
Now changed to using a string as suggested.